Gemfile

```diff
@@ -41,5 +58,7 @@ group :development, :test do
   gem 'web-console', '~> 2.0'
```

Comment:
They are in the group for development. I'm not sure why the '@@ -41,5 +58,7 @@' is showing up in this pull request